# PlayStation 4 Dual Shock Controller Input Mapping
axes: 6
buttons: 12

left: 0
right: 0
up: 1
down: 1

x_a: 1
o_b: 2
square_x: 0
triangle_y: 3

l1_lb: 4
r1_rb: 5